### DISCUSSION TITLE
antd-mobile5.20.0，TypeError: undefined is not a function

### DISCUSSION THREAD

**Discussion by AnneLuc** [ant-design/ant-design-mobile#6652] [Q&A]
求大佬指点，报错信息”
../node_modules/antd-mobile/es/utils/native-props.js in startsWith at line 20:12
“if (key.startsWith('data-') || key.startsWith('aria-')) {”
主要集中在vivo低端机型中

**Comment by zombieJ** [comment]
信息太少了，可以考虑打个断点看看是哪个 props 传空了

**Reply by AnneLuc** [reply]
组件引用是正常的，没有传空的，而且报错主要集中在vivo低端机型中，看着像是antd源码中startWith方法不适配
